apps/wq_bwa/wq_bwa

```diff
@@ -128,10 +128,12 @@ sub split_query {
 		$line_count++;
 	}
 
+    $num_outputs++;
```

Comment:
I would not add this one, and instead use $i <= $num_splits;  in line 144.

Reply:
Getting rid of that makes all the print statements be off by one. Which I guess isn't such a big deal, but still. Do you still want me to change?

Reply:
I see what you mean.